src/maidsafe/vault/mpid_manager/mpid_manager.proto

```diff
@@ -1,4 +1,4 @@
-/*  Copyright 2012 MaidSafe.net limited
+/*  Copyright 2015 MaidSafe.net limited
```

Comment:
If it's newly created then 2015.  However, from this commit it doesn't look like it's new.  I guess a copy-and-paste error from when it was created :-)